charts/snapshot-validation-webhook/README.md

```diff
@@ -1,5 +1,8 @@
 # snapshot-validation-webhook
 
+> **Discontinuing notice**: The chart has been merged with the [*snapshot-controller*](../snapshot-controller/README.md) chart
+> in the same repository. This chart has been archived.
+
 Deploys the [snapshot-validation-webhook](https://github.com/kubernetes-csi/external-snapshotter/#validating-webhook)
```

Comment:
Helm also seems to support a `deprecated:` keyword in Chart.yaml. We should set that and bump the version to trigger a new release. Presumably users would then get a warning about the deprecation.